packages/components/src/DateTimePickers/InputDateTimeRangePicker/__snapshots__/InputDateTimeRangePicker.component.test.js.snap

```diff
@@ -226,7 +226,7 @@ exports[`InputDateTimeRangePicker should render 1`] = `
                             <span
                               style="font-weight: 400; font-size: 1.6rem; visibility: hidden; position: absolute;"
                             >
-                              00:00:00
+                              HH:mm:ss
```

Comment:
changes in this snapshot file are because now we only use placeholder text to measure the width of input.